changelogs/2024-10-03_live-configuration-and-data-editing.mdx

```diff
@@ -0,0 +1,84 @@
+
+You can enable live data and configuration editing on your existing Sites using the *Enable live configuration and data reloading* flag on the *Flags* page. For more information, please read steps one and two of our [Unified Configuration Migration Guide](/documentation/guides/unified-configuration-migration-guide/).
+
+### The Unified Configuration file format
+
+* The Unified Configuration file format for your CloudCannon configuration file simplifies Site setup.
+* When you create a Site using the Unified Configuration file format, CloudCannon requires less information from you. It also uses the same configuration options regardless of SSG.
+* CloudCannon can now detect your Collections immediately after syncing your files, regardless of SSG.
+* You can now manage your content in CloudCannon before your first build.
+* Loading your Sites in CloudCannon is now faster.
+* CloudCannon now provides suggestions for your build configuration settings based on your files and SSG, including your command line options and environment variables.
+* You can now [enable Headless Mode](/documentation/articles/enable-headless-mode/) if you want to create a headless Site on CloudCannon. Headless mode disables building and hosting for your Site and streamlines your *Site Settings* by removing settings you don't need.
+  * Headless Mode is useful if you want to create fully hosted static websites with visual editing, server-side rendering (SSR) or other server-side functions, or fully hosted APIs while still using CloudCannon as your primary CMS.
+* The *Site Details* page now includes fields for your *Site Name*, *Mode*, and *Static Site Generator*. Mode determines whether your Site is Hosted or Headless, toggling Site hosting and builds. Adding a Site icon is now part of the onboarding flow.
+* The *Build Configuration* page now includes a checkbox for *Building Locked* and sections for *Command line options*, *Build system options*, and *Library versions*. The SSG dropdown has moved to the *Site Details* page.
+  * Fields available under *Command line options* are *Install Command*, *Build Command*, *Output Path*, and *Environment Variables*.
+  * Fields available under *Build system options* are *Preserved Paths*, *Preserve Output*, *Include Git Folder*, and *Manually Configure URLs*.
+  * Fields available under *Library versions* are *Hugo version*, *Ruby version*, *Node version*, and *Deno version*.
+  * Many configuration options have been removed from this page as you no longer need to specify them in CloudCannon for Sites using the Unified Configuration file format.
+
+All new Organizations now use the Unified Configuration file format by default.
+
+You can migrate to the Unified Configuration file format on your existing Sites by following our [Unified Configuration Migration Guide](/documentation/guides/unified-configuration-migration-guide/).
+
+## Fixes
```

Comment:
```suggestion
You can migrate to the Unified Configuration file format on your existing Sites by following our [Unified Configuration Migration Guide](/documentation/guides/unified-configuration-migration-guide/).

### Changes to input naming conventions

* New flags have been added to Sites to control some input naming conventions. Existing Sites will be unaffected.
  * The *Infer Select values from input key* flag controls whether inputs will select from collections automatically. For example, whether an input named `author` will automatically become a select box referencing a collection named `authors`. New Sites using Unified Configuration will **not** have this flag enabled, but it can be enabled in the Site settings.
  * The *Infer Structure from input key* flag controls whether inputs will reference a configured structure automatically. For example, whether an input named `content_blocks` will automatically become an array of the structure named `content_blocks`. All sites will continue to have this flag enabled by default, but this can be disabled in the Site settings.
  

## Fixes
```